src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -114,6 +118,8 @@ internal bool TryExport(T data)
                 return true; // enqueue succeeded
             }
 
+            this.ParentProvider?.GetSdkHealthReporter()?.BatchExportProcessorDroppedCount.Add(1, this.tags);
```

Comment:
Probably better perf to pass `TagList` by ref (eg `Add(1, in this.tags)`).

Reply:
![image](https://user-images.githubusercontent.com/17655898/227388342-86ad3888-984f-49cf-ad6b-514bf23e36c4.png)

Reply:
updated to array